src/main/java/com/terraforged/mod/server/command/TerraCommand.java

```diff
@@ -80,9 +82,17 @@
 @Mod.EventBusSubscriber(bus = Mod.EventBusSubscriber.Bus.FORGE)
 public class TerraCommand {
 
+    private static HashMap<UUID, Integer> searchId;
```

Comment:
This can be something like: `private static final Map<UUID, Integer> searchIds = new HashMap<>();` as it is not dependent on the mod initialization lifecycle